package/src/utils/utils.ts

```diff
@@ -52,6 +57,31 @@ export const MessageStatusTypes = {
   SENDING: 'sending',
 };
 
+export type ValueOf<T> = T[keyof T];
+
+export type RequiredFileStates = Omit<typeof FileState, 'NO_FILE'>;
+export type FileStateType = ValueOf<RequiredFileStates>;
+
+export type Progress = ValueOf<typeof ProgressIndicatorTypes>;
+export type IndicatorStatesMap = Record<FileStateType, Progress>;
+export type ProgressIndicatorState = Progress | null;
+
+export const getIndicatorTypeForFileState = (fileState: FileStateType): ProgressIndicatorState => {
```

Comment:
```suggestion
export const getIndicatorTypeForFileState = (fileState: FileState): Progress | null => {
```

you should not assert that only omitted file states are coming in. You are returning null as the default too, and this small change makes it clear.

Reply:
✅